src/main/java/org/folio/oaipmh/domain/Verb.java

```diff
@@ -1,29 +1,24 @@
 package org.folio.oaipmh.domain;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
 import static com.google.common.collect.ImmutableSet.of;
 import static java.util.Collections.EMPTY_SET;
 import static java.util.stream.Collectors.toSet;
 import static java.util.stream.Stream.concat;
-import static org.folio.oaipmh.Constants.FROM_PARAM;
-import static org.folio.oaipmh.Constants.IDENTIFIER_PARAM;
-import static org.folio.oaipmh.Constants.METADATA_PREFIX_PARAM;
-import static org.folio.oaipmh.Constants.RESUMPTION_TOKEN_PARAM;
-import static org.folio.oaipmh.Constants.SET_PARAM;
-import static org.folio.oaipmh.Constants.UNTIL_PARAM;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import static org.folio.oaipmh.Constants.*;
 
 /**
  * Enum that represents OAI-PMH verbs with associated http parameters and validation logic.
  */
 public enum Verb {
-  GET_RECORD("GetRecord", of(IDENTIFIER_PARAM, METADATA_PREFIX_PARAM), EMPTY_SET, null),
+  GET_RECORD("GetRecord", of(IDENTIFIER_PARAM, METADATA_PREFIX_PARAM, EXPIRATION_DATE_RESUMPTION_TOKEN), EMPTY_SET, null),
```

Comment:
You are wrong here, you have to put the EXPIRATION_DATE_RESUMPTION_TOKEN to the third constructor parameter. 
EMPTY_SET -> of(EXPIRATION_DATE_RESUMPTION_TOKEN)